frontend/components/core/ReSearchBar.vue

```diff
@@ -60,12 +59,9 @@ export default {
   border-radius: 5px;
   @include font-size(14px);
   .svg-icon {
-    fill: $primary-color;
+    fill: $font-secondary;
```

Comment:
The core search bar changes also styles because the  rules management view ? This will affect to all uses of searchbar

Reply:
the rules new design includes some little changes in commons components